api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java

```diff
@@ -1363,30 +1512,58 @@ public static CohortDefinition getMQ13DenB5_P4(
     return compositionCohortDefinition;
   }
 
+  public static CohortDefinition getMQ13DenB5_PregnantCV50(
```

Comment:
There is no need to have this method, since you can call the inner method directly where you need it to be implemented

Reply:
Done